src/components/navbar/navbar.js

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import "./navbar.styles.css";
+
+function Navbar({ logoImage }) {
+  const [toggleShowNavbarNav, setToggleShowNavbarNav] =  useState("");
+  const { logoPath, logoDesc } = logoImage;
+
+  function hamburgerMenuClickHandler(){
+        setToggleShowNavbarNav(prevShow => prevShow ? "" : "show");
+  }
+
+  return (
+    <nav className="navbar">
+      <div className="navbar-brand">
+        <a href="#" className="nav-link">
+          <img
+            src={`${process.env.PUBLIC_URL}${logoPath}`}
+            alt={logoDesc}
+            width="40"
+            height="40"
+            className="d-inline-block align-text-top"
+          />
+          <span className="estore-name">3 on 3 Store</span>
+        </a>
+      </div>
+
+      <div className="search-box">
```

Comment:
Search box can be itself be a component.

Reply:
Suggestion registered. :)